src/content/blog/drupalcon-barcelona-2024.md

```diff
@@ -0,0 +1,64 @@
+
+* **Working together: DDEV on Gitpod and GitHub Codespaces**: We talked about the many ways to use DDEV in Cloud IDEs like Gitpod and GitHub Codespaces. People were really interested in this, since of course these approaches don't require a local development environment at all (or rather, the local environment is in the cloud on a VM, and presented via a browser). See the DDEV docs on [Gitpod](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-gitpod) and [GitHub Codespaces](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-codespaces). 
+* **Git on the Command Line - Office Hours**: We had fun talking about the basics of Git, what people need to know to really master it and do their work predictably and without fear. This one was the first thing in the morning, so had a smaller turnout.
+* **DDEV Office Hours**: At most DrupalCons we do this simple free-form BoF so that people can bring the issues and discussions they want to have. At this one we had some really strong and experienced DDEV users, so it ended up being that they were helping me to understand their workflow rather than me coaching them.
+* **Prepare for Contribution Day: Issue Forks, Drupalpod, DDEV, Git**: The highlight of DrupalCon is the Contribution Day, but sometimes coming to it it's a little intimidating because of Drupal's unusual contribution workflow. We talked about Drupal's issue workflow, how issue forks work, merge requests, etc. In the Drupal workflow, each issue has its own Git project fork, instead of the traditional open-source workflow where each person has a personal fork. In the Drupal workflow, anybody can push to any of the branches, or create a new branch. It's quite amazing.
+
+## Mentoring on Contribution Day
+
+The highlight of the week is when lots of people spend the day working on specific contributions. I normally help mentor first-time contributors on that day, helping with workflow, DDEV, Gitpod, and Drupalpod concerns. As usual, it was great and we got to work with lots of first-time folks, and many were successful understanding the workflow for the first time. One of the new contributors actually got to have their merge request merged in a live pull on-screen. But a few more got all the way to RTBC (Reviewed and Tested By the Community).
+
+## Mauricio Dinarte Session on DDEV
+
+[Mauricio Dinarte](Local development environments for Drupal with DDEV) did a great exposition of the features of DDEV in [Local development environments for Drupal with DDEV](https://events.drupal.org/barcelona2024/session/local-development-environments-drupal-ddev). I didn't even know this was coming up, but I was amazed at what he was able to introduce people to in 45 minutes. When the recording becomes available I'll promote it on the socials, because it was great.
```

Comment:
```suggestion
[Mauricio Dinarte](https://www.drupal.org/u/dinarcon) did a great exposition of the features of DDEV in [Local development environments for Drupal with DDEV](https://events.drupal.org/barcelona2024/session/local-development-environments-drupal-ddev). I didn't even know this was coming up, but I was amazed at what he was able to introduce people to in 45 minutes. When the recording becomes available I'll promote it on the socials, because it was great.
```